nb/projects/gear/gear.nb

```diff
@@ -19,5 +19,4 @@ const example = Gear()
   .hasToothResolution(toothResolution)
   .ex(thickness)
   .material('wood')
-  .view(1024)
```

Comment:
I know this is a deleted line, but what does the argument to view() do?

Reply:
It was a size at one time. :)

Reply:
Ah right, I remember that now.